src/main/resources/com/google/api/codegen/php/discovery_fragment.snip

```diff
@@ -248,8 +248,8 @@
   optParams
 @end
 
-@private requestBodyName()
-  requestBody
+@private postBodyName()
+  postBody
```

Comment:
Per offline discussion, let's keep our original name and rewrite it in the compile checker like the one for Python.